src/main/java/com/pingcap/tikv/types/DurationType.java

```diff
@@ -0,0 +1,18 @@
+package com.pingcap.tikv.types;
```

Comment:
Is this conflict with Sam's PR? Please remove this if so.

Reply:
Conflict, removed.

Reply:
Conflict, removed.